File: hospitalManagement/EmergencyTransportation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace hospitalManagement
{
    internal abstract class EmergencyTransportation : MovementEquipment
    {
        public enum DangerousLevel
        {
            Minor,
            Moderate,
            Considerable,
            High,
            VeryHigh
        }
        //Field
        private DangerousLevel level;
        // Infor field
        //  Management field
        // Dynamic field
        // Properties
        public DangerousLevel Level
        {
            get => level; set => level = value;
        }
        // Constructors
        public EmergencyTransportation() { }
        public EmergencyTransportation(

             string id
            , string name
            , string description
            , float price
            , bool state
            , string origin
            , int quantity
            , int numberOfSeat
            , string destination
            , DangerousLevel level
            ) : base(id, name, description, price, state, origin, quantity, numberOfSeat, destination)
        {
            this.Level = level;
        }
        // Destructors
        ~EmergencyTransportation() { }
        // Methods
        // in, output
        public override void Input()
        {
            base.Input();
            Console.Write("Dangerous level ( 0.Minor, 1.Moderate, 2.Considerable, 3.High, 4.Very high): ");
            this.Level = (DangerousLevel)(Int32.Parse(Console.ReadLine()));
        }
        public override void Output()
        {
            base.Output();
            Console.WriteLine($"Dangerous level : {Level}");
        }
        // General method
        // Other method
        // Overriding
        public override string ToString()
        {
            return base.ToString()
                + $"\nDangerous level: {Level}";
        }
    }
}

File: src/hospitalManagement/EmergencyTransportation.java
package hospitalManagement;

import java.util.Scanner;

public abstract class EmergencyTransportation extends MovementEquipment {

    public enum DangerousLevel
    {
        Minor,
        Moderate,
        Considerable,
        High,
        VeryHigh
    }
   
	//Field
    private DangerousLevel level;
    // Infor field
    //  Management field
    // Dynamic field
    // Properties
    public DangerousLevel getLevel() {
		return level;
	}
	public void setLevel(DangerousLevel level) {
		this.level = level;
	}
    // Constructors
    public EmergencyTransportation() { }
    public EmergencyTransportation(

         String id
        , String name
        , String description
        , float price
        , boolean state
        , String origin
        , int quantity
        , int numberOfSeat
        , String destination
        , DangerousLevel level
        ) 
    {
    	super(id, name, description, price, state, origin, quantity, numberOfSeat, destination);
        this.level = level;
    }
    // Destructors

    // Methods
    // in, output
    public  void Input()
    {
    	Scanner ip = new Scanner(System.in);
        super.Input();
        System.out.print("Dangerous level ( 0.Minor, 1.Moderate, 2.Considerable, 3.High, 4.Very high): ");
        int choice = ip.nextInt();
        switch (choice) {
        case 0:
        	this.level = DangerousLevel.Minor;
        	break;
        case 1:
        	this.level = DangerousLevel.Moderate;
        	break;
        case 2:
        	this.level = DangerousLevel.Considerable;
        	break;
        case 3:
        	this.level = DangerousLevel.High;
        	break;
        case 4:
        	this.level = DangerousLevel.VeryHigh;
        	break;
        }
        ip.close();
    }
    public  void Output()
    {
        super.Output();
        System.out.println("Dangerous level : " + level);
    }
    // General method
    // Other method
    // Overriding
    public  String ToString()
    {
        return super.toString()
            + "\nDangerous level: " + level;
    }
}
